src/app/containers/PageHandlers/withVariant/index.jsx

```diff
@@ -0,0 +1,37 @@
+import pathOr from 'ramda/src/pathOr';
+import React from 'react';
+import { useParams, useLocation, Redirect } from 'react-router-dom';
+import getVariantRedirectUrl from './getVariantRedirectUrl';
+
+const WithVariant = Component => {
+  const VariantContainer = props => {
+    const { service, variant } = useParams();
+    const location = useLocation();
+
+    const { path, params } = pathOr({}, ['match'], props);
```

Comment:
I think `match` either needs a better name or some description as to what it does and what it should look like

Reply:
match is a default object we get from react-router

Reply:
ok cool didn't know that. can we explicitly pass in to the HOC what we want from it then?

Reply:
Fixed!

Reply:
I think we'd need a parent HOC if we wanted to do that. The match prop comes from react-router.

Reply:
ah ok no probs